src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1394 @@
+                + capacity
+                * feed_conc_ppm
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][7]
+                + capacity
+                * b.recovery_vol_phase[0, "Liq"]
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][8]
+                + capacity
+                * feed_temperature
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][9]
+                + motive_pressure
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][10]
+                + motive_pressure
+                * feed_conc_ppm
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][11]
+                + motive_pressure
+                * b.recovery_vol_phase[0, "Liq"]
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][12]
+                + motive_pressure
+                * feed_temperature
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][13]
+                + motive_pressure
+                * capacity
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][14]
+                + 1 * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][15]
+                + motive_pressure**2
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][16]
+                + capacity**2
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][17]
+                + feed_temperature**2
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][18]
+                + b.recovery_vol_phase[0, "Liq"] ** 2
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][19]
+                + feed_conc_ppm**2
+                * motive_steam_mass_flow_rate_coeffs[b.number_effects.value][20]
+            )
+
+        # Energy consumption
+        @self.Constraint(doc="Specific thermal energy consumption calculation")
+        def eq_specific_energy_consumption_thermal(b):
+            return b.specific_energy_consumption_thermal == pyunits.convert(
+                1
+                / b.gain_output_ratio
+                * (
+                    b.motive_steam_props[0].enth_mass_phase["Vap"]
+                    - b.heating_steam_props[0].enth_mass_phase["Liq"]
+                )
+                * b.distillate_props[0].dens_mass_phase["Liq"],
+                to_units=pyunits.kWh / pyunits.m**3,
+            )
+
+        @self.Constraint(doc="Thermal power requirement calculation")
+        def eq_thermal_power_requirement(b):
+            return b.thermal_power_requirement == pyunits.convert(
+                b.specific_energy_consumption_thermal
+                * b.distillate_props[0].flow_vol_phase["Liq"],
+                to_units=pyunits.kW,
+            )
+
+        # Mass flow rate
+        @self.Constraint(doc="Feed and cooling water mass flow rate (kg/s)")
```

Comment:
This looks more like an enthalpy balance to me. Maybe you should rename the constraint, change comment above, and change doc string if that is the case.

Reply:
Doc string, constraint name and comment updated.